### PR TITLE
Remove screen lesson, rewrite data flow chapters

### PR DESCRIPTION
**Pull request by alexpearce**
This is towards issue #137. I've reordered the table of contents, rewrote the data flow lesson, added a lesson on the data in run two, and removed the lesson on `screen`.

I haven't gone through any of the other lessons. In an upcoming PR I'll add references to the story at the beginning and end of each lesson.

So, the two data flow lessons need a read through to check for not-stupid-ness.

### CODE REVIEW

File: 07-dataflow.md [hunk 1]
```diff
@@ -1,54 +1,151 @@
...
+    objects such as tracks and clusters. This is done by the Brunel 
+    application. The objects are stored into an output file in a ‘DST’ format.
+ 3. The reconstructed DST files are suitable for analysis, but they are not 
+    accessible to users due to computing restrictions. Data are filtered 
+    further through a set of selections called the *stripping*, controlled by the 
+    DaVinci application which write output either in DST or ‘µDST’ (micro-DST) 
+    format. To save disk space and to speed up access for analysts, stripping 
+    selections that select similar types of decays, such as those containing fully 
+    hadronic decays of charm hadrons and those containing a dimuon pair, there a 
+    _streams_ which contain similar selections.
```
Comment (betatim) [inline 07-dataflow.md]: woah, can we shorten the last sentence? Also missing a verb (grouped into)

**Comment by betatim** [comment]
I like it, in particular the intro `19-...md`.

**Comment by betatim** [comment]
Should we search and replace Brunel -> `Brunel` etc?

**Comment by alexpearce** [comment]
Thanks for the feedback @betatim!

I actually changed “`Brunel`” to “Brunel”. It's the name of a piece of software, not a variable of code snippet, so it shouldn't be typeset in monospace.

File: 07-dataflow.md (hunk 1 above)
Comment (alexpearce) [inline 07-dataflow.md]: For sure. Hows about this:

> The reconstructed DST files are suitable for analysis, but they are not 
> accessible to users due to computing restrictions. Data are filtered 
> further through a set of selections called the _stripping_, controlled by 
> the DaVinci application which write out data either in the DST or ‘µDST’ 
> (micro-DST) format. To save disk space and to speed up access for analysts, the 
> output files are grouped into _streams_ which contain similar selections.
> By grouping all of the fully hadronic charm selections together, for example, 
> analysts interested in that type of physics don't waste time running of the 
> output over the output of the dimuon selections.

Comment (betatim) [inline 07-dataflow.md]: "of the output over the output" -> "over the output of" otherwise :+1:

Comment (alexpearce) [inline 07-dataflow.md]: Quite right. I've pushed the changes.

**Comment by alexpearce** [comment]
Is this good to go? Can someone merge?

**Comment by betatim** [comment]
Merging

**Pull request merged**